recipe/meta.yaml

```diff
@@ -43,41 +39,43 @@ outputs:
         - python
         - pip
         - certifi >=2020.06.20
+        - freetype >=2.3
+        - numpy
+        - setuptools
         - setuptools_scm >=4
         - setuptools_scm_git_archive
-        - numpy
-        - cycler >=0.10
-        - python-dateutil >=2.7
-        - freetype
-        - nose
-        - pyparsing >=2.2.1
         - tk  # [linux]
       run:
         - python
-        - setuptools
-        - {{ pin_compatible('numpy') }}
-        - certifi >=2020.06.20
         - cycler >=0.10
         - fonttools >=4.22.0
-        - python-dateutil >=2.1
+        - freetype >=2.3
+        - kiwisolver >=1.0.1
+        - {{ pin_compatible('numpy') }}
+        - packaging >=20.0
         - pillow >=6.2.0
-        - freetype
         - pyparsing >=2.2.1
+        - python-dateutil >=2.7
         - tk  # [linux]
-        - tornado
-        - kiwisolver >=1.0.1
+        # Installing from a git checkout.
+        - setuptools_scm >=4
```

Comment:
`conda` users will not be installing from a git checkout, so we do not need to include this in the run requirements.

```suggestion
```